js/imageLoad.js

```diff
@@ -4,7 +4,7 @@ function loadJSON(callback) {
         xobj.overrideMimeType("application/json");
     xobj.open('GET', 'js/logos.json', true);
     xobj.onreadystatechange = function () {
-          if (xobj.readyState == 4 && xobj.status == "200") {
```

Comment:
Not sure about this though. It's possible a user's connection is bad and the json couldn't be fetched properly by the browser.

Reply:
Well my reason for removing the status check is:
1. It works only on server [Voiding the site from running locally]
2. Since what its actually checking for is a "JSON file"[That always exists and is accessible i.e. 200 at all times] there is expressly no need for it.

Reply:
See what i'm talking about. 

Regardless of whether the file is on the server or not, as a user, your browser downloads the file.

That can fail for any reason. The status might not always be 200.

<img width="1267" alt="screen shot 2019-02-28 at 10 11 40" src="https://user-images.githubusercontent.com/13024935/53555083-8615cf00-3b41-11e9-8602-4a3e0aa7426f.png">

Reply:
I'm aware. My opinion is that if the status is not 200 for some reason then the logos won't load [Nothing breaks, nothing is affected]. 
The typical next thing is the user reloads the page.

Reply:
Touche. That's fine den.